From f13d46c91ee70303972cfbc3b612aa70898829b8 Mon Sep 17 00:00:00 2001
From: Scott K Logan <logans@cottsay.net>
Date: Fri, 22 Mar 2024 20:42:47 -0500
Subject: [PATCH] Install the package.xml manifest

---
 rpm/ros-rolling-iceoryx-binding-c.spec | 1 +
 1 file changed, 1 insertion(+)

diff --git a/rpm/ros-rolling-iceoryx-binding-c.spec b/rpm/ros-rolling-iceoryx-binding-c.spec
index 3b9b559c2f24..d25f4683238b 100644
--- a/rpm/ros-rolling-iceoryx-binding-c.spec
+++ b/rpm/ros-rolling-iceoryx-binding-c.spec
@@ -63,6 +63,7 @@ Eclipse iceoryx inter-process-communication (IPC) middleware C-Language Binding
 
 %install
 %cmake_install
+install -m0644 -p -D package.xml %{buildroot}%{bloom_prefix}/share/%{pkg_name}/package.xml
 
 
 %if 0%{?with_tests}
-- 
2.44.0

