From f00732b9646abe6fbc031bba4beddfc1816bd8ac Mon Sep 17 00:00:00 2001
From: Scott K Logan <logans@cottsay.net>
Date: Fri, 22 Mar 2024 20:42:47 -0500
Subject: [PATCH] Install the package.xml manifest

---
 rpm/ros-rolling-iceoryx-introspection.spec | 1 +
 1 file changed, 1 insertion(+)

diff --git a/rpm/ros-rolling-iceoryx-introspection.spec b/rpm/ros-rolling-iceoryx-introspection.spec
index 9e32c19fe364..9097d94d91c2 100644
--- a/rpm/ros-rolling-iceoryx-introspection.spec
+++ b/rpm/ros-rolling-iceoryx-introspection.spec
@@ -69,6 +69,7 @@ client
 
 %install
 %cmake_install
+install -m0644 -p -D package.xml %{buildroot}%{bloom_prefix}/share/%{pkg_name}/package.xml
 
 
 %if 0%{?with_tests}
-- 
2.44.0

